### ISSUE TITLE
Determine which Mining script we will be using

### ISSUE THREAD

**Issue by mnkolev** [mnkolev/charcoal#2]
- Will we be incorporating an existing script into our packaged software?
- Will we be making any changes to existing scripts before incorporating?

**Comment by tpedelose** [comment]
If starting with Ether (imo, we should) I'd recommend Ethminer. It's Open Source (pretty sure) and pretty stable now.

**Comment by mnkolev** [comment]
We also need to keep watching to see if theres any on the rise we can jump on in the process.

**Comment by tpedelose** [comment]
Ideally we add multiple miners to the platform and let users choose up to X% utilization of their card(s) for one currency and Y% for another.

**Comment by tpedelose** [comment]
From what I've seen they're all pretty similar. The two biggest ones are Ethminer and Claymore. Claymore is alot like what we are trying to do but NOT open source. The creator takes 1% computing power for its use mining ETH and another 1% for dual mining ETH and SIA.

**Comment by mnkolev** [comment]
Yeah we should definitely do Ethminer and then modify our overall system to take a small percentage but using the open source miner should be fine as it is
